### PR TITLE
Flow fixes

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core Please review

**Comment by anmolbabu** [comment]
TODO: Need to fix etcd_orm usage @ time of updating atom execution status to etcd

File: tendrl/commons/jobs/__init__.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def _process_job(self, raw_job, job_key):
             # Generate a request ID for tracking this job
             # further by tendrl-api
             req_id = str(uuid.uuid4())
-            if tendrl_ns.type == "node":
+            if tendrl_ns.type == "node" or tendrl_ns.type == "monitoring":
```
Comment (r0h4n) [inline tendrl/commons/jobs/__init__.py]: how many types does performance and alerts repo define? this is something the tendrl-api decides, please clarify @anmolbabu 

cc @anivargi

Comment (anmolbabu) [inline tendrl/commons/jobs/__init__.py]: I didn't know about this. I just need one type to target all monitoring jobs to the monitoring module

Is there a detailed spec or a mail thread that gives an idea of :
* what each of these fields mean/serve?
* The conventions followed etc....

Comment (r0h4n) [inline tendrl/commons/jobs/__init__.py]: "type=monitoring" is ok then. youll need another type for the alerts?

Comment (anmolbabu) [inline tendrl/commons/jobs/__init__.py]: No not required for alerting as of now

**Pull request merged**